scripts/commitlint/index.js

```diff
@@ -0,0 +1,51 @@
+const { Octokit } = require('@octokit/rest')
+const lint = require('@commitlint/lint')
+
+const { rules } = require('../../commitlint.config')
+
+function getPrNumber() {
+  const prUrl = process.argv[2]
+  const prUrlParts = prUrl.split('/')
+
+  return prUrlParts[prUrlParts.length - 1]
+}
+
+function getCommitsByPr(number) {
+  const github = new Octokit()
+  return github.pulls.listCommits({
+    owner: 'Royal-Navy',
```

Comment:
Are we able to get `owner` and `repo` from somewhere too so it doesn't need to be hardcoded?